tests/unit/test_exceptions.py

```diff
@@ -275,31 +275,56 @@ def create_bad_request_details():
     return status_detail
 
 
+def create_error_info_details():
+    info = error_details_pb2.ErrorInfo(
+        reason="SERVICE_DISABLED",
+        domain="googleapis.com",
+        metadata={
+            "consumer": "projects/455411330361",
+            "service": "translate.googleapis.com",
+        },
+    )
```

Comment:
This bit is likely the source of the lint failure on Kokoro.  Please run `nox -s blacken lint` and re-push.

Reply:
Done